Makefile

```diff
@@ -120,10 +120,18 @@ LDFLAGS := $(shell source ./hack/version.sh; version::ldflags)
 ## --------------------------------------
 
 # The number of ginkgo tests to run concurrently
-E2E_GINKGO_PARALLEL=2
+E2E_GINKGO_PARALLEL ?= 2
 
 E2E_ARGS ?=
 
+E2E_GINKGO_FOCUS ?=
+E2E_GINKGO_SKIP ?=
+
+# to set multiple ginkgo skip flags, if any
+ifneq ($(strip $(E2E_GINKGO_SKIP)),)
+_SKIP_ARGS := $(foreach arg,$(strip $(E2E_GINKGO_SKIP)),-skip="$(arg)")
```

Comment:
for now ,there is no SKIP test , you will add the E2E full test into this skip list (so it will not run every PR
but should be daily/weekly or manual trigger?)

Reply:
Actually my plan was to use focus to run the default tests (`E2E_GINKGO_FOCUS="\[PR-Blocking\]"`). Then for the e2e full test we would skip PR-Blocking since it would anyway run on all PRs. This is the same way that CAPI has done it.

Here CAPI focus on PR-Blocking for their default e2e test: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml#L120-L150
And here they skip PR-Blocking and other tests that would anyway run separately: https://github.com/kubernetes/test-infra/blob/b27527630f6d4c0f2787f0362efd54c4cab5fbd6/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml#L222-L256